show-mod.php

```diff
@@ -1,185 +1,345 @@
+		latest unstable version that is either
+			for the unstable version of the game, or
+			for the stable version of the game, if the release is a newer unstable version than the stable release (only if there is no release for the unstable game version)
+		-> recommend for testers
+		If there are not releases for for either of these, select the latest release -> latest release for an outdated version of the game
+
+	Examples assuming current game version = 5, an a newer unstable version = 5p, GV = game version, RV = mod release version:
+		GV  RV
+		5   2.5
+		5   3
+		5   4.1  -> Recommended
+		5p  4.2  -> For testers
+
+		5   2.5
+		5   3    -> Recommended
+		5   4.p1
+		5   4.p2 -> For testers
+
+		5   2.5
+		5   3    -> Recommended
+		5   4.p1
+		5p  4.p2 -> For testers
+
+		2   2.5
+		2   3
+		3   4.p1
+		3   4.p2 -> Latest outdated
+
+		Assuming we came herey by searching for mods for GV 2:
+		2   2.5
+		2   3    -> Recommended*
+		3   4.p1
+		3   4.p2
+*/
+
+/*
+	NOTE(Rennorb): The mod list/search should pass information about the currently searched for game versions to this script, so we can recommend the correct release when user is searching with a specific gv in mind.
+	This could be accomplished in one of three ways:
+		Post parameter:
+			pros:
+			- Invisible (does not pollute url)
+			cons:
+			- Causes the browser to query for "are you sure you want to resend information" when the page gets reloaded.
+			- Does not get transferred over if the link gets copy pasted to another user, potentially causing confusion because the site displays something different for the other user.
+		Get parameter:
+			pros:
+			- Does get copied over to other users, preserving that specific recommendation in the process.
+			cons:
+			- Pollutes the page link when it gets copy pasted to be presented on some social media outlet -> discord mod showcase links would likely get polluted.
+			- Recommendation is pinned with this link -> stored links 'degrade' since they won't recommend the latest release but the one for the specified game version.
+		Cookie:
+			pros:
+			- Invisible (does not pollute url)
+			cons:
+			- Potential misinterpretation if the cookie is not correctly reset before the user navigates to the mod page without a specific version search.
+			- Does not get transferred by copying the link, potentially causing confusion in a third party that will get a different recommendation.
+			- Likely does not persist between page reloads, so reloading the page will reset the recommendation to the general recommendation.
+		Referrer:
+			pros:
+			- Invisible (does not pollute url)
+			- Persistent across reloads, but resets on navigation.
+			- Does not require javascript.
+			cons:
+			- Does not get transferred by copying the link, potentially causing confusion in a third party that will get a different recommendation.
+			- Sending a correct referrer is up to the client.

```

Comment:
another downside: Sharing a mod link will show different downloads for the other users

Reply:
one more: Should we ever add a list of dependencies / recommended mods to a mod view page then these would also not show the right version

Reply:
line 242 already states that, no? 
but yea, is definitely the main issue with the invisible approaches